### PR TITLE
Solution of language basics-1

### PR DESCRIPTION
**Pull request by tavisca-tkatyal**
@tavisca-vtripathi

### CODE REVIEW

**Comment by tavisca-tkatyal** [comment]
@tavisca-vtripathi

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 1]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            int prev = equation.IndexOf('*');
+            string A = equation.Substring(0,prev);
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:26]: use a proper naming convention. Rename `**A**` to a meaningful variable name

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 2]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
+            int next = equation.IndexOf('=');
+            string B = equation.Substring(prev+1,next-prev-1);
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:28]: Use the proper naming convention. 
 Rename **B**

https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 3]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
+            string C = equation.Substring(next+1);
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:29]: Same as above

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 4]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:24]: **`Create another Class FixMultiplication and create new method FindDigit there.`**

 And from Class Program's method Find Digit, call the method of FixMultiplication

**Review by tavisca-vtripathi — CHANGES_REQUESTED** [review]

**Comment by tavisca-tkatyal** [comment]
@tavisca-vtripathi 
Changes done

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 5]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
+            int prev = equation.IndexOf('*');
+            string A = equation.Substring(0,prev);
+            int next = equation.IndexOf('=');
+            string B = equation.Substring(prev+1,next-prev-1);
+            string C = equation.Substring(next+1);
+            //Console.WriteLine($"{A} {B} {C}");
+            if(C.Contains('?'))
+            {
+                float a = float.Parse(A);
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:33]: what is float a? how it is different from A?

Can give some meaningful name to it.

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 6]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            int prev = equation.IndexOf('*');
+            string A = equation.Substring(0,prev);
+            int next = equation.IndexOf('=');
+            string B = equation.Substring(prev+1,next-prev-1);
+            string C = equation.Substring(next+1);
+            //Console.WriteLine($"{A} {B} {C}");
+            if(C.Contains('?'))
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:31]: We are checking the same conditions in If and elseif. can we put it in a method and do the calling from here?

File: Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs [hunk 7]
```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
...
+            {
+                float a = float.Parse(A);
+                float b = float.Parse(B);
+                float res = a*b;
```
Comment (tavisca-vtripathi) [inline Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs:35]: user complete name, not an abbreviation.

**Review by tavisca-vtripathi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**